toolchain/lc-compile/src/check.g

```diff
@@ -1754,8 +1754,6 @@
         Id'Position -> Position
 		Id'Namespace -> Namespace
         (|
-			where(Namespace -> id(_))
-		||
```

Comment:
@runrevmark, I think that in places where identifiers are qualified, then they don't count as reserved identifiers.  For example, when I write `com.livecode.canvas`, that shouldn't trigger a "reserved identifier" warning, but when I write `variable __naughty`, it should.  It looks like the builds are failing because the compiler is incorrectly emitting warnings for module names.